frontend/server/k8s-helper.ts

```diff
@@ -62,6 +62,62 @@ function getNameOfViewerResource(logdir: string): string {
   return 'viewer-' + crypto.SHA1(logdir);
 }
 
+function parseTensorboardLogDir(logdir: string, podTemplateSpec: object): string {
```

Comment:
Can we add a comment above? the purpose of this function is to support `volume://` url in logdir, otherwise, there's no need to parse logdir, we can just use them.